### DISCUSSION TITLE
Repository Configuration as Secrets

### DISCUSSION THREAD

**Discussion by parasappafluke** [argoproj/argo-cd#6634] [Q&A]
Hi,
As documented in https://argoproj.github.io/argo-cd/operator-manual/declarative-setup/ i have created **_argocd-repositories.yaml_** and **_argocd-repo-creds.yaml_** files as per the spec provided in the example. I applied this to argocd ns using **_kubectl apply_** command. In the UI i couldn't see any repositories and credentials template added. Below is my sample credentials template and repo secrets.
**ArgoCD Version:** v2.0.4

**Repository Secret:**
`apiVersion: v1
kind: Secret
metadata:
  name: test-repo-name
  namespace: argocd
  labels:
    argocd.argoproj.io/secret-type: repository
stringData:
  url: https://github.com/testOrg/test-repo-name`

**Repo Credential Template Secret:**

`apiVersion: v1
kind: Secret
metadata:
  name: testOrg-https-creds
  namespace: argocd
  annotations:
    managed-by: argocd.argoproj.io
  labels:
    argocd.argoproj.io/secret-type: repo-creds
type: Opaque
stringData:
  url: https://github.com/testOrg
  password: ****
  username: ****`

**Comment by jannfis** [comment]
Duplicate of #6637
